src/plotter.py

```diff
@@ -11,12 +11,29 @@
 
 
 class Plotter:
-    def __init__(self, data, n_age: int) -> None:
+    def __init__(self, data, n_age: int, model: str) -> None:
         self.data = data
         self.n_age = n_age
-        self.labels = ["0-4", "5-9", "10-14", "15-19", "20-24", "25-29",
-                       "30-34", "35-39", "40-44", "45-49", "50-54", "55-59",
-                       "60-64", "65-69", "70-74", "75+"]
+        if model in ["rost", "kenya"]:
```

Comment:
Create a JSON file where you put these lists, and load that JSON, and based on model name you can simply fill `self.labels`.